DbMocker/MockTableFromType.cs

```diff
@@ -11,12 +11,14 @@ public partial class MockTable
         /// <summary />
         public static MockTable FromType<T>(
             IEnumerable<T> rows = null,
-            BindingFlags propertyBindingFlags = DefaultFromTypeBindingFlags
+            BindingFlags propertyBindingFlags = DefaultFromTypeBindingFlags,
+            string[] columnsToInclude = null
         )
         {
             var propertyInfos = typeof(T).GetProperties(propertyBindingFlags);
 
             var columns = propertyInfos
+                .Where(propertyInfo => columnsToInclude == null || columnsToInclude.Contains(propertyInfo.Name))

```

Comment:
well thats a great question. since its a whitelist yeah, but on the other hand this would be pretty useless. we can treat empty like null for that matter

Reply:
Yes, probably better to use:
```csharp
.Where(propertyInfo => columnsToInclude == null || columnsToInclude.Length == 0 || columnsToInclude.Contains(propertyInfo.Name))
```